site/content/concepts/device-notifications.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Device notifications (from 0.6 on)"
+weight = 415
++++
+
+*Business applications* can be informed by Hono about specific states of a device. This concept - refered to as
+ **Device notifications** - is described in detail here.
+ 
+ 
+
+{{% warning %}}
+This concept is currently part of active development and discussion.
+The content here summarizes the current discussion and is being prototyped.
+Please join our mailing list or consult the IRC channel in case you have any questions or want to participate.
+{{% /warning %}}
+
+<!--more-->
+
+For implementing specific behaviour *Business applications* need to know if
+
+- a device is currently ready to receive a command
+- a device is currently connected to a protocol adapter
```

Comment:
I was under the impression that we wanted to separate these two concerns. So I propose to remove the *connected to protocol adapter* stuff.